en/docs/extended_topics/model_mixed_parallel.md

```diff
@@ -1,12 +1,12 @@
-# Characteristics of parallelism in OneFlow
+# Features of parallelism in OneFlow
 
-In [Consistent and Mirrored view](consistent_mirrored.md), we have already known OneFlow provides mirrored and consistent two point of view and learned about the  `consistent` in OneFlow have some special characteristics.
+In [Consistent and Mirrored view](consistent_mirrored.md), we have already known OneFlow provides mirrored and consistent two point of view and learned about the  `consistent` view in OneFlow have some special features.
```

Comment:
we have already known OneFlow provides mirrored and consistent two point of view
->
we have already known OneFlow provides two types of view: mirrored and consistent view